### ISSUE TITLE
[Organizer Request] Add Natalia Granato to Juiz de Fora chapter

### ISSUE THREAD

**Issue by ste-bartels** [cncf/communitygroups#250]
### Name of chapter inquiring about

Cloud Native Juiz de Fora

### Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io

Tell us about yourself. List your name, email, company, GitHub URL, LinkedIn URL, and CNCF Slack workspace name (or if you need to be invited), and a required profile link on community.cncf.io

Name: Natália Granato
Email: granatonatalia@gmail.com
Company: Magalu Cloud
Github: https://github.com/nataliagranato
Linkedin: https://www.linkedin.com/in/nataliagranato/

### Tell us why you want to be added or removed from the chapter

Natalia has extensive experience with the community, being a cncf ambassador in Brazil. It will bring great recognition to our local community.

### Code of Conduct

- [x] I agree to follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)

### Where organizer resides

- [x] I confirm I am local to or live near the city of the above-proposed chapter (do not mark if not relevent to request)

### Organizers companies/affiliations

- [x] I come from xyz company which is differing from that of the current organizers (do not mark if not relevent to request)

### Inclusive

- [x] I agree. To assure this, I have recently completed the [Inclusive Open Source Community Orientation (LFC102) course](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/) (do not mark if not relevent to request)

### Vendor Neutral

- [x] I have read what [vendor-nuetrality means](https://contribute.cncf.io/maintainers/community/vendor-neutrality/), and I agree the main content of our meetups will remain vendor-nuetral (do not mark if not relevent to request)

### Remain an active chapter

- [x] I understand the activity level standards of all CNCF chapters (do not mark if not relevent to request)

### Collaboration with other chapters

- [x] I acknowledge the CNCF right to approve additional chapters in my area based on this criteria (do not mark if not relevent to request)

### Aim to have a diverse representation of speakers in your meetups

- [x] I will do my best to host diverse speakers throughout the year (do not mark if not relevent to request)

**Comment by nataliagranato** [comment]
@AudMonte01 I have recently completed the Inclusive Open Source Community Orientation (LFC102) course: https://www.credly.com/badges/2ada49c7-36c0-4f58-88bb-3f6618beaeff/public_url

**Comment by pacheco-bs** [comment]
I agree. Natália is deeply committed and consistently delivers valuable contributions that drive the growth and success of the community.

**Comment by ste-bartels** [comment]
I completely agree! Natália is incredibly engaged and plays a key role in fostering the community's growth and success

**Comment by phacerqueira** [comment]
I wholeheartedly agree! Natália is deeply involved and plays an essential part in driving the community's development and achievements.

**Comment by AudMonte01** [comment]
Thank you everyone! @nataliagranato - I have added you as an organizer. Please accept my apologies for the delay past the 30-day mark.

You should have just received an automated welcome email from me with next steps. And if you agree to the above organizer terms, please check the box upon accessing the chapter via Bevy.

Should you have any questions, you can always join my office hours on Fridays. The times vary slightly: https://community.cncf.io/community-event-organizer-hub/

This issue is now closed.
